CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
+
+FEATURES:
+* **New Resource:** `google_apigee_environment_keyvaluemaps_entries` ([#7717](https://github.com/hashicorp/terraform-provider-google-beta/pull/7717))
+* **New Resource:** `google_apigee_environment_keyvaluemaps` ([#7717](https://github.com/hashicorp/terraform-provider-google-beta/pull/7717))
+* **New Resource:** `google_compute_resize_request` ([#7725](https://github.com/hashicorp/terraform-provider-google-beta/pull/7725))
+* **New Resource:** `google_compute_router_route_policy` ([#7748](https://github.com/hashicorp/terraform-provider-google-beta/pull/7748))
+* **New Resource:** `google_scc_v2_organization_mute_config` ([#7744](https://github.com/hashicorp/terraform-provider-google-beta/pull/7744))
+
+IMPROVEMENTS:
+* alloydb: added `observability_config` field to `google_alloydb_instance` resource ([#7737](https://github.com/hashicorp/terraform-provider-google-beta/pull/7737))
+* bigquery: added `resource_tags` field to `google_bigquery_dataset` resource (ga) ([#7718](https://github.com/hashicorp/terraform-provider-google-beta/pull/7718))
+* bigquery: added `resource_tags` field to `google_bigquery_table` resource ([#7735](https://github.com/hashicorp/terraform-provider-google-beta/pull/7735))
+* bigtable: added `data_boost_isolation_read_only` and `data_boost_isolation_read_only.compute_billing_owner` fields to `google_bigtable_app_profile` resource ([#7789](https://github.com/hashicorp/terraform-provider-google-beta/pull/7789))
+* cloudfunctions: added `build_service_account` field to `google_cloudfunctions_function` resource ([#7713](https://github.com/hashicorp/terraform-provider-google-beta/pull/7713))
+* compute: added `aws_v4_authentication` field to `google_compute_backend_service` resource ([#7775](https://github.com/hashicorp/terraform-provider-google-beta/pull/7775))
+* compute: added `custom_learned_ip_ranges` and `custom_learned_route_priority` fields to `google_compute_router_peer` resource ([#7727](https://github.com/hashicorp/terraform-provider-google-beta/pull/7727))
+* compute: added `export_policies` and `import_policies` fields  to `google_compute_router_peer` resource ([#7748](https://github.com/hashicorp/terraform-provider-google-beta/pull/7748))
+* compute: added `shared_secret` field to `google_compute_public_advertised_prefix` resource ([#7767](https://github.com/hashicorp/terraform-provider-google-beta/pull/7767))
+* compute: added `storage_pool` under `boot_disk.initialize_params` to `google_compute_instance` resource ([#7787](https://github.com/hashicorp/terraform-provider-google-beta/pull/7787))
+* compute: changed `target_service` field on the `google_compute_service_attachment` resource to accept a `ForwardingRule` or `Gateway` URL. ([#7736](https://github.com/hashicorp/terraform-provider-google-beta/pull/7736))
+* container: added documentation for `reservation_affinity` field of `google_container_node_pool` resource ([#7741](https://github.com/hashicorp/terraform-provider-google-beta/pull/7741))
```

Comment:
Is this a documentation only change?

Reply:
[Yeah](https://github.com/GoogleCloudPlatform/magic-modules/pull/11161/files) - I'll remove via a suggestion!